sdk/__tests__/utils.ts

```diff
@@ -431,9 +470,7 @@ async function deployEvm(ctx: Ctx): Promise<Ctx> {
 
   // Setup the initial calls, like transceivers for the manager
   console.log("Set transceiver for manager");
-  await tryAndWaitThrice(() =>
-    transceiver.getAddress().then((addr) => manager.setTransceiver(addr))
-  );
+  await tryAndWaitThrice(() => manager.setTransceiver(transceiverProxyAddress));
```

Comment:
This has been changed form the transceiver address to the proxy address. 

I think this is correct but I'm not positive